newTest.ts

```diff
@@ -0,0 +1,138 @@
+const users : [{"id": Number,"name": String}] = require("./data/users"); //Import users.json in data folder. 'require' will always load the JSON data as a JavaScript object
+const express = require("express"); //Import the express module into the file
+const app = express(); 
+app.use(express.json()); //express.json() is a middleware function that parses JSON payload, if any, in the incoming API requests
+const fs = require("fs");//To use the file system operation 
+
+//http://localhost:3000/api/users
+app.get('/api/users',(req,res)=>{ //Get all the user from the list
+    res.send(users);
+});
+
+//http://localhost:3000/api/user/1
+app.get('/api/user/:id',(req,res)=>{  //Get the specific user from the list
```

Comment:
Always validate the user input first before processing the request. Here the user could have sent a string or special character instead of a number.